apps/mkt/components/features-slack.tsx

```
@@ -0,0 +1,144 @@
+import FeatureImage01 from "@/public/images/home.png";
+import DigestMetrics from "@/public/images/digest-metrics.png";
+import DigestWip from "@/public/images/digest-wip.png";
+import { IconChartBar, IconProgress } from "@tabler/icons-react";
+
+export default function FeaturesSlack() {
+  const [tab, setTab] = useState<number>(1);
+
+  const tabs = useRef<HTMLDivElement>(null);
+
+  const heightFix = () => {
+    if (tabs.current && tabs.current.parentElement)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)